tmlib/workflow/illuminati/api.py

```diff
@@ -148,16 +148,17 @@ def create_run_batches(self, args):
                             if layer.channel.bit_depth == 8:
                                 if clip_max < 255:
                                     clip_max = 255
+                                    
+                            clip_min = stats.get_closest_percentile(0.001)
                             else:
-                                if clip_max < 700:
-                                    clip_max = 700
+                                if clip_max < clip_min + 255:
```

Comment:
This code belongs to the `else:` branch of `if layer.channel.bit_depth == 8`, so we know that `layer.channel.bit_depth > 8`, right?  So why constrain `clip_max - clip_min == 255`?  Shouldn't we be able to use more than 256 levels of gray here?